students/student_framllat/session03/slicer.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+def exch_first_last(n):
+
+    """ This will swap first/last element in sequence
+    """
+
+    a = list(n)
+    a[0], a[-1] = a[-1], a[0]
+    if type(a[0]) is str:
+        return ''.join(a)
+    elif type(a[0]) is int:
+        return a
+    else:
+        return None
+
+def rm_every_oth(n):
+
+    """ This will remove every other element in sequence
+    """
+
+    a = list(n)
+    del a[1::2]
```

Comment:
instead of deleting -- what if you had returned a new sequence with every other element still there?